CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Current Master
 - Readded loose keyring label back into the account list.
+- Added button to reject all transactions (thanks [davidp94](https://github.com/davidp94)!)
```

Comment:
I prefer no markdown in changelogs, since it's used to generate text that we post on various mediums, that don't all support markdown.